config/settings/test.yml

```diff
@@ -1,2 +1,2 @@
 processing_thread_pool: 5
-hathi_overlap_path: spec/fixtures/hathitrust/
+hathi_overlap_path: spec/fixtures/hathitrust/overlap.tsv
```

Comment:
very minor, not important but renaming it as to sth like `hathi_overlap_file` would it make more sense?

Reply:
isn't that implied by the directory it is in?